python/ray/scripts/scripts.py

```diff
@@ -193,13 +193,19 @@ def cli(logging_level, logging_format):
     "--temp-dir",
     default=None,
     help="manually specify the root temporary dir of the Ray process")
+@click.option(
+    "-f",
+    "--config-file",
+    default=None,
+    type=str,
+    help="If specified, use config options from this file. ")
```

Comment:
remove trailing white space and make this say

`"Do NOT use this. This is for debugging and development purposes ONLY."`